### PR TITLE
create a helper class for setblock flags

### PR DESCRIPTION
**Pull request by SpaceWalkerRS**

### CODE REVIEW

File: src/main/java/net/nullspace_mc/tapestry/mixin/feature/fillupdates/SetBlockCommandMixin.java [hunk 1]
```diff
@@ -24,6 +24,11 @@ public abstract class SetBlockCommandMixin extends Command {
     )
     private boolean disableUpdate(World world, int x, int y, int z, Block block, int metadata, int flags) {
         SetBlockHelper.applyFillUpdatesRule = true;
-        return world.setBlockWithMetadata(x, y, z, block, metadata, Settings.fillUpdates ? 3 : 2);
+
+        if (!Settings.fillUpdates) {
+            flags &= ~SetBlockFlags.UPDATE_NEIGHBORS;
```
Comment (SRAZKVT) [inline src/main/java/net/nullspace_mc/tapestry/mixin/feature/fillupdates/SetBlockCommandMixin.java]: This uses non-intuitive at first glance operators, can we have a comment ? something along the lines of 

```
flags &= ~SetBlockFlags.UPDATE_NEIGHBORS; // turn off update neighbors flag
```

**Review by SRAZKVT — CHANGES_REQUESTED** [review]
Fill and Clone commands should be affected by the fillUpdates rule

**Comment by SpaceWalkerRS** [comment]
> Fill and Clone commands should be affected by the fillUpdates rule

Considering that that's a bug with the `fillUpdates` rule rather than this change, should I fix that in a separate PR?

**Comment by SRAZKVT** [comment]
> Considering that that's a bug with the `fillUpdates` rule rather than this change, should I fix that in a separate PR?

sure

File: src/main/java/net/nullspace_mc/tapestry/mixin/feature/fillupdates/SetBlockCommandMixin.java (hunk 1 above)
Comment (SpaceWalkerRS) [reply src/main/java/net/nullspace_mc/tapestry/mixin/feature/fillupdates/SetBlockCommandMixin.java]: done

**Comment by SRAZKVT** [comment]
looks good to me

**Pull request merged**